File: 01 - Microservices/01 - Introduccion/Microservices.GraphQL/Microservices.GraphQL.Orders.Service/Models/Order.cs
using Microservices.GraphQL.Orders.Service.GraphQL;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Microservices.GraphQL.Orders.Service.Models
{
    public class Order
    {        
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int Id { get; set; }
        public int ProductId { get; set; }
        public int Quantity { get; set; }
        public string Status { get; set; } 
        public double TotalPrice { get; set; }        
    }

}

File: 01 - Microservices/01 - Introduccion/Microservices.gRPC/ng-microservices-grpc-client/src/app/models/order.ts
export class Order {
  constructor() {
    this.orderId = '';
    this.productName = '';
    this.status = '';
    this.totalPrice = 0;
   }

  public orderId: string;
  public productName: string;
  public status: string;
  public totalPrice: number;  
}
